tests/pytests/test_wildcard.py

```diff
@@ -387,3 +387,26 @@ def testSuffixCleanup(env):
   conn.execute_command('DEL', 'doc')
 
   forceInvokeGC(env, 'idx')
+
+def testMOD7453():
+  """Tests that we don't enter an infinite loop when we match a wildcard to a
+    wildcard in the matched term"""
+
+  env = DialectEnv()
+  conn = getConnectionByEnv(env)
+
+  # Create an index with a TEXT and TAG field
+  env.cmd('FT.CREATE', 'idx', 'SCHEMA', 'tag', 'TAG')
```

Comment:
Also test with `TEXT`?

Reply:
Also add to [unit test](https://github.com/RediSearch/RediSearch/blob/cd4469dcf1ad9832cdca24368248033cfe44c58c/tests/ctests/test_wildcard.c#L174C5-L174C15)?

Reply:
I tested locally and there were no problems
I can add some here as well